example/mbed-os-5/mbed_app.json

```diff
@@ -0,0 +1,12 @@
+{
+    "macros": [
+        "MEM_ALLOC=malloc",
+        "MEM_FREE=free"
+    ],
+    "target_overrides": {
+        "*": {
+            "target.features_add": ["COMMON_PAL"],
```

Comment:
`"platform.stdio-baud-rate": 115200,` here should be enough.

`"platform.stdio-convert-newlines": true,`, and possibly even `"platform.stdio-buffered-serial": true,` might be worth configuring too, unless the idea is to have the absolute bare minimum.